src/SfxWeb/src/app/views/cluster/essentials/essentials.component.ts

```diff
@@ -91,6 +98,17 @@ export class EssentialsComponent extends BaseControllerDirective {
       this.nodes.refresh(messageHandler).pipe(map(() => {this.updateItemInEssentials(); })),
       this.systemApp.refresh(messageHandler).pipe(catchError(err => of(null))),
       this.clusterUpgradeProgress.refresh(messageHandler),
+      this.RestClient.getPartitionById("00000000-0000-0000-0000-000000000001", messageHandler).pipe(map((partition) => {
+        this.fmQuorumLossStatus = partition.PartitionStatus;
+        if (this.fmQuorumLossStatus == "InQuorumLoss") {
+          this.fmQuorumLossWarning = "The Failover Manager service is in quorum loss state. " +
+          "Cluster may not be responding. " + 
+          "Service failover, automatic recovery will be blocked. " + 
+          "Health/availability state of services/nodes may not be reflected. "
```

Comment:
Why are you using + and multiple strings here? If you just want to break up the string so it doesn't look too long, you can use [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals), which allows you to break a string into multiple lines. Also, since this string is not used anywhere else and does not need to have any other values, you can initialize it with this value and remove the if statement.